misc/config_tools/schema/checks/check_the_checkers.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Should put a copyright and SPDX header comment in scripts too.

Reply:
Added